bin2c/bin2c.c

```diff
@@ -29,6 +29,9 @@
 #include <stdlib.h>
 #include <string.h>
 
+#define H_OUTPUT_FORMAT_STR	"0x%0.2X"
```

Comment:
Whitespace after the define name not needed.

Reply:
> Whitespace after the define name not needed.

Oops, did not see that tab on my editor. I'll fix it.